doc/devel/documenting_mpl.rst

```diff
@@ -111,6 +111,12 @@ it, use
 .. code-block:: sh
 
    make SPHINXOPTS= html
+   
+On Windows the arguments must be at the end of the statement:
+
+.. code-block:: bat
+
+   make html SPHINXOPTS=
```

Comment:
```suggestion
   make.bat html SPHINXOPTS=
```
Not tested but I suppose, you want the .bat file.

Reply:
no, cmd.exe will happily append it for you